andela/travel_tool_back
# #162127512 Seeders file for Approvals and Trips

#### What does this PR do?
It contains seeders for approvals and Trips

#### Description of Task to be completed?
- It contains seeders file for approvals and Trips

#### How should this be manually tested?
- Clone the Travela backend repo `git clone https://github.com/andela/travel_tool_back.git`
- Checkout to this branch `git checkout ch-manager-request-approval-e2e-test-162127512`
- Run command to make ssh `make ssh`
- Run the command below on your terminal to rollback, migrate and seed e2e data to your database
`yarn db:rollback && yarn db:migrate && yarn db:seed:e2e`
- Check your database to confirm that the Trips table, Approval tables have been populated with data.

#### Any background context you want to provide?
- None

#### What are the relevant pivotal tracker stories?
[#162127512](https://www.pivotaltracker.com/story/show/162127512)

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
